src/utils/ServerTime.js

```diff
@@ -0,0 +1,60 @@
+import { getUTCEpoch, getLocalDate, getUTCDate } from './index';
+
+class ServerTime {
+    clockStarted = false;
+
+    constructor(api) {
+        this._api = api;
+        this.init();
+    }
+
+    init() {
+        if (!this.clockStarted) {
+            clearInterval(this.getTimeInterval);
+            this.requestTime();
+            this.getTimeInterval = setInterval(this.requestTime.bind(this), 60000);
```

Comment:
why are you updating every 60 seconds? Binary-static updates every 30 seconds.

Reply:
@brucebinary I wanted to reduce the server requests but 30 sec is okay too.